site/js/modules/webui/server.js

```diff
@@ -167,11 +167,27 @@ function renderEndpointArea(endpoints, selectedEndpoint, context) {
 function callEndpoint(targetEndpoint, inputFields, context, container) {
     Routing.loadingStart(container.querySelector(".results-area"), false);
 
+    let override_email = undefined;
+    let override_cleartext = undefined;
+
     let params = {};
+    let clearContextUser = true;
     for (let field of inputFields) {
```

Comment:
I would put a newline above this (since you are no longer just populating `params`).